### ISSUE TITLE
R.objFromKeys(valueFunction, keys)

### ISSUE THREAD

**Issue by asaf-romano** [ramda/ramda#1124]
Similar to mapObj, a utility that generates an object given a value-for-key function and an array of keys seems useful.

Something that works like this: http://bit.ly/1AwyGiv

**Comment by CrossEye** [comment]
I agree that this is useful.  We've already seen a number of implementations.  I would guess that a `reduce`-based one would end up being both more performant and easiest to understand.

Do you feel like creating a PR?

**Comment by asaf-romano** [comment]
I'll try to do so over the weekend. Need to catch up with Haskell-docs and github mechanics first :)

**Comment by raine** [comment]
This would be a great inclusion, not sure about the name though. It sounds like it'd map keys of an object.

**Comment by lwassermann** [comment]
I agree that the name might be misleading.

For some reason this reminds me of `into`. But when trying to replicate the behavior with `into`, the [result](http://bit.ly/1LR5cMF) is rather hard to follow.

**Comment by buzzdecafe** [comment]
@lwassermann that's pretty nice, i think. it's amazing how many different impls people have come up with for this problem

**Comment by asaf-romano** [comment]
Another naming option: fromKeys (analogy to fromPairs)

**Comment by raine** [comment]
objFromKeys, mapWithKeys

Sent from my iPhone

--
Raine
